elastalert/elastalert.py

```diff
@@ -460,15 +460,27 @@ def get_hits_count(self, rule, starttime, endtime, index):
 
     def get_hits_terms(self, rule, starttime, endtime, index, key, qk=None, size=None):
         rule_filter = copy.copy(rule['filter'])
-        if qk:
+        qk_list = qk.split(", ")
```

Comment:
I think this will fail if qk is not provided, since it will be None object. We probably need to keep the ``if qk:`` if statement.